MongoLabKit/Sources/Classes/MongoLabError.swift

```diff
@@ -9,22 +9,22 @@
 import Foundation
 
 public enum MongoLabError: ErrorDescribable {
-    case ConnectionError
-    case RequestError
-    case ParserError
-    case ServerError(statusCode: Int, message: String?)
+    case connectionError
+    case requestError
+    case parserError
+    case serverError(statusCode: Int, message: String?)
 
 
     public func description() -> String {
         switch self {
-        case .ConnectionError:
+        case .connectionError:
             return "Unable to connect to server."
-        case .RequestError:
+        case .requestError:
             return "Unable to create a request to the database. baseURL and apiKey MUST be not empty in the configuration."
```

Comment:
Not sure if these are user facing but perhaps we can bundle these into a Localisable.strings bundle to be embedded into the framework. Not related to the Swift 3 migration though